pkgs/os-specific/linux/zfs/unstable.nix

```diff
@@ -12,18 +12,19 @@ in
 callPackage ./generic.nix args {
   # check the release notes for compatible kernels
   kernelCompatible = if stdenv'.isx86_64 || removeLinuxDRM
-    then kernel.kernelOlder "6.4"
+    then kernel.kernelOlder "6.5"
     else kernel.kernelOlder "6.2";
 
-  latestCompatibleLinuxPackages = linuxKernel.packages.linux_6_1;
+  latestCompatibleLinuxPackages = linuxKernel.packages.linux_6_4;
```

Comment:
You are not doing the distinction for aarch64, are you?

Reply:
Is there still one? I don’t know the status of ARM…